config/locales/en/settings.yml

```diff
@@ -207,3 +207,14 @@ en:
         max_size_description: "Maximum size allowed for an uploaded document (in Megabytes/MB)"
         content_types: "Accepted content types for documents"
         content_types_description: "Select all the content types allowed for uploaded documents"
+    sdg:
+      process_debates: Linking debates to SDG
+      process_debates_description: Enable debates to be linked to Sustainable Development Goals
```

Comment:
A couple of suggestions:

1. Allow debates to be linked to Sustainable Development Goals
2. Make it possible to link debates to Sustainable Development Goals